src/dmd/traits.d

```diff
@@ -1920,7 +1920,7 @@ Lnext:
         return tup.expressionSemantic(sc);
     }
 
-    extern (D) const(char)[] trait_search_fp(const(char)[] seed, ref int cost)
+    static const(char)[] trait_search_fp(const(char)[] seed, ref int cost)
```

Comment:
You shouldn't change the function linkage
```suggestion
    extern (D) static const(char)[] trait_search_fp(const(char)[] seed, ref int cost)
```

Reply:
it's extern D already.

Reply:
Someone has set it as `extern(D)` because it used to be `extern(C++)` but just removing the attribute declaration would have be enough, [see blame](https://github.com/dlang/dmd/blame/767efb7ec57db1e4a21552d3ad7d039b77616a0d/src/traits.d#L308).